### ISSUE TITLE
Orca on Linux does not announce "formatting" or "saving"

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#196487]
Testing #196126

* Set `accessibility.alert.format` and/or `accessibility.alert.save` to always
* Start Orca on Linux
* 🐛 I don't hear the screenreader announce that it's formatting or saving

**Comment by meganrogge** [comment]
Are you in screen reader mode?

**Comment by TylerLeonhardt** [comment]
Yes I had 

```
"editor.accessibilitySupport": "on"
```

**Comment by paulacamargo25** [comment]
The same thing happens to me on macOS, a sound plays, but I don't hear the screen reader say "formatted".

**Comment by meganrogge** [comment]
When you hear an audio cue, you will by design NOT hear "formatted"

**Comment by meganrogge** [comment]
Either a sound will play or you'll hear the action that happened, never both.

**Comment by TylerLeonhardt** [comment]
Ah... that's what this was. I do hear "formatted" and "saved" when I turn the audio cues off

**Comment by paulacamargo25** [comment]
i turned the audio cues off, but i still can't hear the accessibility alerts.

**Comment by meganrogge** [comment]
Which screen reader are you using and what are your settings @paulacamargo25 ?

**Comment by paulacamargo25** [comment]
Sorry it was my fault. I had disabled the screen reader. Now it works, thanks :)

**Comment by jvesouza** [comment]
In my environment every time I press ctrl+s to save a file, I hear the saved message, even if the file does not need to be saved. I don't know if this is the expected behavior. It seems to me that the ideal would be to hear the message only if the file was actually saved.

**Comment by meganrogge** [comment]
@jvesouza IMO, that is by design, as we do save the document whether or not it's needed when a user uses `ctrl+s`.
